addons/binding/org.openhab.binding.airvisualnode/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="airvisualnode"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="avnode">
+		<label>AirVisual Node</label>
+		<description>AirVisual Node air quality monitor</description>
+
+		<!-- Channel groups -->
+
+		<channel-groups>
```

Comment:
`channel-groups` are for making it possible to re-use a group of channels on several `thing-types` not to actually group them by same characteristics. so please use plain channels.